apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -200,3 +200,42 @@ function wpcom_track_global_styles( $blog_id, $post, $updated ) {
 	}
 }
 add_action( 'save_post_wp_global_styles', 'wpcom_track_global_styles', 10, 3 );
+
+/**
+ * Adds the global style notice banner to the custom launch bar controls.
+ *
+ * @param array $custom_controls List of custom controls.
+ *
+ * return array The collection of launch bar custom controls to render.
+ */
+function wpcom_display_global_styles_banner( $custom_controls ) {
+	// Do not show the banner if the user can use global styles.
+	if ( ! wpcom_should_limit_global_styles() ) {
```

Comment:
We should also bail if the site doesn't have any style hidden (e.g. a site that doesn't use a block theme, or a site with a block theme that's using the default variation).

Reply:
The site can not use a block theme and still have custom styles saved in the Global Styles CPT.
What do you think if we implement it as a follow-up? I think it's going to be pretty complex to determine all those conditions.

Reply:
Created a follow-up task for this. https://github.com/Automattic/wp-calypso/issues/68643

Reply:
I'd be ok with deploying this change without addressing that, but I don't think we can release this to end-users without tackling some very likely conditions.

(We're probably on the same page, but just trying to be explicit)

Reply:
I'm fine leaving that for a follow-up.